docs/internals/data-structures.rst

```diff
@@ -615,13 +615,34 @@ with data and seeking over the empty hole ranges).
 +++++++++++++++++
 
 The buzhash chunker triggers (chunks) when the last HASH_MASK_BITS bits of the
-hash are zero, producing chunks with a target size of 2^HASH_MASK_BITS Bytes.
+hash are zero, producing chunks with a target size of 2^HASH_MASK_BITS bytes.
 
 Buzhash is **only** used for cutting the chunks at places defined by the
 content, the buzhash value is **not** used as the deduplication criteria (we
 use a cryptographically strong hash/MAC over the chunk contents for this, the
 id_hash).
 
+The idea of content-defined chunking is assigning every byte where a
+cut *could* be placed a hash. The hash is based on some number of bytes
+(the window size) before the byte in question. Chunks are cut
```

Comment:
Hmm, maybe talking about `offsets` instead of `bytes` would make it even more clear?

Reply:
Hmm, considering that content might shift to other offsets, speaking of bytes might be better (as they don't change, but offsets of them do).